assets/wizards/popups/views/campaigns/index.js

```diff
@@ -0,0 +1,301 @@
+			),
+		} ) )
+	);
+	grouped.push( {
+		label: __( 'Default (no segment)', 'newspack' ),
+		id: '',
+		prompts: prompts.filter( ( { options: { selected_segment_id: segment } } ) => ! segment ),
+	} );
+	return grouped;
+};
+
+const dataForCampaignId = ( id, campaigns ) =>
+	campaigns.reduce( ( acc, group ) => ( +id > 0 && +id === +group.term_id ? group : acc ), null );
+
+/**
+ * Campaign management screen.
+ */
+const Campaigns = props => {
+	const {
+		campaignId,
+		prompts = [],
+		campaigns = [],
+		manageCampaignGroup,
+		segments,
+		wizardApiFetch,
+		refetch,
+		duplicateCampaignGroup,
+		deleteCampaignGroup,
+		archiveCampaignGroup,
+		renameCampaignGroup,
+	} = props;
+
+	const [ popoverVisible, setPopoverVisible ] = useState();
+	const [ modalVisible, setModalVisible ] = useState();
+	const [ modalType, setModalType ] = useState();
+	const [ campaignName, setCampaignName ] = useState();
+	const [ inFlight, setInFlight ] = useState( false );
+
+	const history = useHistory();
+
+	const submitModal = modalText => {
+		if ( MODAL_TYPE_NEW === modalType ) {
+			createTerm( modalText );
+		} else if ( MODAL_TYPE_RENAME === modalType ) {
+			renameCampaignGroup( campaignId, modalText );
+		} else if ( MODAL_TYPE_DUPLICATE === modalType ) {
+			duplicateCampaignGroup( campaignId, modalText );
+		}
+		setModalVisible( false );
+	};
+
+	const createTerm = term => {
+		setPopoverVisible( false );
+		setInFlight( true );
+		wizardApiFetch( {
+			path: '/wp/v2/newspack_popups_taxonomy',
+			method: 'POST',
+			quiet: true,
+			data: {
+				name: term,
+				slug: term,
+			},
+		} )
+			.then( () => {
+				setInFlight( false );
+				setCampaignName( '' );
+				refetch();
+			} )
+			.catch( () => {
+				setInFlight( false );
+			} );
+	};
+
+	const activeCampaigns = campaigns.filter( ( { status } ) => 'archive' !== status );
+	const archivedCampaigns = campaigns.filter( ( { status } ) => 'archive' === status );
+	const campaignData = dataForCampaignId( campaignId, campaigns );
+
+	const valueForCampaignId = id => {
+		if ( 'unassigned' === id ) {
+			return {
+				key: 'unassigned',
+				name: __( 'Unassigned Prompts', 'newspack' ),
+			};
+		}
+
+		const data = dataForCampaignId( id, campaigns );
+
+		if ( data ) {
+			return {
+				key: data.term_id,
+				name: data.name,
+			};
+		}
+		return {
+			key: 'active',
+			name: __( 'Active Prompts', 'newspack' ),
+		};
+	};
+	return (
+		<Fragment>
+			<div className="newspack-campaigns__campaign-group__filter-group-wrapper">
+				<div className="newspack-campaigns__campaign-group__filter-group-actions">
+					<CustomSelectControl
+						label={ __( 'Campaigns', 'newspack' ) }
+						options={ [
+							{ key: 'active', name: __( 'Active Prompts', 'newspack' ) },
+							{ key: 'unassigned', name: __( 'Unassigned Prompts', 'newspack' ) },
+							{
+								key: 'header-campaigns',
+								name: __( 'Campaigns', 'newspack' ),
+								className: 'is-header',
+							},
```

Comment:
When there are no Campaigns, the UI looks like it's missing something – perhaps a "No Campaigns found" message could be displayed in the dropdown?

![image](https://user-images.githubusercontent.com/7383192/106471587-13869a00-64a2-11eb-9a10-e27756231e85.png)

Reply:
Resolved in https://github.com/Automattic/newspack-plugin/pull/833/commits/c6c9d25d743703737e55cf59c069897432351a13 by hiding the `Campaigns` header row if none exist.